packages/fluentui/accessibility/src/behaviors/Tree/treeTitleBehavior.ts

```diff
@@ -22,6 +25,7 @@ const treeTitleBehavior: Accessibility<TreeTitleBehaviorProps> = props => ({
         'aria-setsize': props.treeSize,
         'aria-posinset': props.index,
         'aria-level': props.level,
+        'aria-selected': props.selected : undefined,
```

Comment:
This is not correct syntax, build is failing :(
Should it be just
```suggestion
        'aria-selected': props.selected,
```
?

Reply:
Fixed